sunpy/map/maputils.py

```diff
@@ -54,6 +55,17 @@ def all_coordinates_from_map(smap):
     return smap.pixel_to_world(x, y)
 
 
+def all_corner_coords_from_map(smap):
```

Comment:
Somewhere along the way you accidently pulled extra commits. You will want to update any local remotes and rebase this pull request.

Reply:
sure @nabobalis i will look into it. Sorry for late response, i am wondering why i didn't get any notification.